Makefile

```diff
@@ -52,7 +52,11 @@ __check_defined = \
 
 .PHONY: snapshot
 snapshot:
-	docker run --rm  -v $(PWD):/workdir -w /workdir cbroglie/goreleaser-cgo:1.12.12-musl goreleaser --rm-dist --snapshot --skip-publish
+	docker run --rm --privileged \
+    -v $(PWD):/cross \
+    -v /var/run/docker.sock:/var/run/docker.sock \
```

Comment:
Is the `--privileged` and access to the docker socket needed? (effectively, this gives full root access to the host)

Reply:
Removed `--privileged` and docker socket from `snapshot` and `github-release` make targets.
Thanks for pointing it out.